src/kinetics/ReactionRateFactory.cpp

```diff
@@ -75,6 +76,11 @@ ReactionRateFactory::ReactionRateFactory()
         return new PlogRate(node, rate_units);
     });
 
+    // LinearBurkeRate evaluator
+    reg("linear-burke", [](const AnyMap& node, const UnitStack& rate_units) {
```

Comment:
Proper nouns in YAML identifiers should be capitalized
```suggestion
    reg("linear-Burke", [](const AnyMap& node, const UnitStack& rate_units) {
```